lib/queue_classic/version.rb

```diff
@@ -0,0 +1,3 @@
+module QC
+  VERSION = "3.1.0RC"
```

Comment:
there is a `3-1-stable` branch already. I think `master` should be version `3.2.0.alpha`